pkg/importer/datastream_ginkgo_test.go

```diff
@@ -11,25 +11,21 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	"kubevirt.io/containerized-data-importer/pkg/image"
-	"kubevirt.io/containerized-data-importer/pkg/importer"
 	imagesize "kubevirt.io/containerized-data-importer/pkg/lib/size"
-	f "kubevirt.io/containerized-data-importer/test/framework"
+	f "kubevirt.io/containerized-data-importer/tests/framework"
```

Comment:
Same, don't think this is right. There are several more of these.

Reply:
The `test` package is being removed and the `framework` package is being moved to `tests`.  This is the reason for the change.